pyproject.toml

```diff
@@ -17,8 +17,12 @@ fastapi = "0.100.1"
 uvicorn = "0.23.2"
 httpx = "0.24.1"
 black = "23.7.0"
-isort = "5.12.0"
+ruff = "0.0.285"
 
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.ruff]
```

Comment:
É bom ler isso aqui: https://beta.ruff.rs/docs/faq/#is-ruff-compatible-with-black